tasks/src/zero-shot-classification/data.ts

```diff
@@ -1,16 +1,35 @@
 import type { TaskDataCustom } from "../Types";
 
 const taskData: TaskDataCustom = {
-	datasets: [],
+
+	datasets: [
+		{
+			description: "",
+			id: "",
+		},
+	],
 	demo: {
 		inputs: [],
 		outputs: [],
 	},
-	metrics: [],
-	
-	models: [],
-	summary:      "Zero-shot text classification is a task in natural language processing (NLP) where a model is trained on a set of labeled examples but is then able to classify new examples from previously unseen classes by transferring knowledge from seen to unseen classes.",
-	widgetModels: [],
+	metrics: [
+		{
+			description:
+				"",
+			id: "",
+		},
+	],
+	models: [
```

Comment:
Can you run github action called "manual lint" so your code is properly linted 🙂

Reply:
hmm the action is failing on the commit step -- would you know by chance why the linter is erroring out? 
  - it fails with "error code 1", I tried running the linting steps locally and got them to execute without a problem -- however no file gets changed after running `npm run format`, is there a different method to lint the code? 

Manual Action Page- https://github.com/ckgresla/hub-docs-zsc/actions/runs/3831405428/jobs/6520494113 (this is where the logs are for the linting action's failure) 

Let me know if there is somewhere else I can look to run the linter and resolve this!